pkg/diagnose/ports/ports.go

```diff
@@ -72,6 +72,15 @@ func DiagnosePortSuite() []diagnosis.Diagnosis {
 			continue
 		}
 
+		if port.Pid == 0 {
```

Comment:
Could you add a comment here describing the OS's behavior regarding the user running the command? Also, is this specific to Debian, and if so please comment about that as well.

Reply:
It was discovered on Debian, but I also replicated the behavior on Ubuntu and RHEL, so I think it's pretty generic. I'll include that info in my comment.